### DISCUSSION TITLE
Access the installed Plausible version easily? Perhaps just add it and show it in the account settings?

### DISCUSSION THREAD

**Discussion by raabmar** [plausible/analytics#4131] [Self-Hosted Support]
It is super complicated to figure out what version I am running Plausible. Could it not be just displayed somewhere in the footer, or in the account settings instead?

**Comment by ruslandoga** [comment]
👋 @raabmar 

Isn't checking the image tag in `docker-compose.yml` or `docker ps` easy enough?

The problem with displaying the version from the container is that BUILD_METADATA JSON is not encoded properly by docker/build-push-action so we don't have that information in Plausible right now: https://github.com/docker/build-push-action/issues/726

**Reply by raabmar** [reply]
I see, I thought it's perhaps just something super simpel to add to. I personally sometimes don't know anymore when doing many other things in life what version I run on

You showed me already once here how to do it but this never worked for me
https://github.com/plausible/analytics/discussions/3554#discussioncomment-7670899

I updated to 2.1.0 today, and coming from 2.1.0. rc 1 (or maybe it was rc 0) it was working without any problems. Zero problems during the update. I had quite many problems with other updates in the last 6 months. So I wanted to check, did it actually work and succeed. Am I actually running on 2.1.0 or am I perhaps still on 2.1.0 rc 1. 

It just comes in handy to show somewhere which version I am on.

I understand if it's not so easy to pull this data in, then it has a super low priority
